### PR TITLE
Label: Smart dex trader

### PR DESCRIPTION
**Pull request by jtxmp**
Brief comments on the purpose of your changes:

"Real" user activity is an important metric for DEX projects to track, in order to advise product strategy and development effectively. This label seeks to identify top retail DEX traders based on heuristics and average cost basis analysis.

**For Dune Engine V2**

I've checked that:

### General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [x] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [x] the filename is unique and ends with .sql
* [x] each sql file is a select statement and has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

File: models/labels/dex/Persona/smart_dex_traders/ethereum/labels_smart_dex_traders_ethereum.sql [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+
+/*
+** In this section, we apply heuristics (H) to filter for retail behavior.
+*/
+
+-- H1. A retail trader does not execute the same amount of trades every day. eg, takes weekends off, prefers mondays.
+--     Trades taken should tend twards weekends or weekdays. ie. skewness <> 0
+, filter_by_weekday as (
+    select taker, 
+        skewness(day_of_week(block_date)) as trading_day_variance
```
[the remark this answers is not in the document]
Comment (jtxmp) [reply models/labels/dex/Persona/smart_dex_traders/ethereum/labels_smart_dex_traders_ethereum.sql:115]: Hi @Hosuke ,

Thanks for the note! 

I was looking into [databricks docs](https://docs.databricks.com/sql/language-manual/functions/dayofweek.html#dayofweek-function) and [spark 3.3.2 docs](https://spark.apache.org/docs/latest/sql-ref-functions-builtin.html#date-and-timestamp-functions:~:text=the%20date/timestamp.-,dayofweek,-(date))

Would it work if we changed it to `dayofweek(block_date)` ?

Thanks for your patience!

[the remark this answers is not in the document]
Comment (Hosuke) [reply models/labels/dex/Persona/smart_dex_traders/ethereum/labels_smart_dex_traders_ethereum.sql:115]: We can try on engine `v2 spark sql` on dune.com first.

[the remark this answers is not in the document]
Comment (jtxmp) [reply models/labels/dex/Persona/smart_dex_traders/ethereum/labels_smart_dex_traders_ethereum.sql:115]: tried on v2 spark sql and it works: https://dune.com/queries/2216119

[the remark this answers is not in the document]
Comment (Hosuke) [reply models/labels/dex/Persona/smart_dex_traders/ethereum/labels_smart_dex_traders_ethereum.sql:115]: You can update in this PR as well

[the remark this answers is not in the document]
Comment (jtxmp) [reply models/labels/dex/Persona/smart_dex_traders/ethereum/labels_smart_dex_traders_ethereum.sql]: Hi @Hosuke , thanks for your help. I've addressed the outstanding issues and both checks are passing now.

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**